woocommerce/woocommerce-ios
# Remove Danger: Dangerfile, gem and CircleCI job

Danger has now been replaced with its self-hosted equivalent, [Peril](https://github.com/danger/peril).

All existing rules from the `Dangerfile` have been added to our new Peril settings repo: https://github.com/wordpress-mobile/peril-settings. This will allow us to share rules across projects and platforms.

You can see an example of Peril failing here: https://github.com/woocommerce/woocommerce-ios/pull/722#issuecomment-467815483

To test:

- See that there is a new Peril check on this PR, and there is no Danger check.
- There is a Hound check on this PR. This replaces `danger-swiftlint`.

Update release notes:
